drivers/adc/mcp356x/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
You can update this Copyright to Launch

Reply:
Do we have a standard for what to attribute to launch vs to the person. I noticed theres a fair mix of Copyright Aaron and Copyright Launch. Should it just always be launch?